topics/spacevim/index.md

```diff
@@ -0,0 +1,16 @@
+---
+aliases: space vim
```

Comment:
Topics themselves can't have spaces in them, so this isn't a valid alias. Can you change this to `aliases: space-vim`? It looks like https://github.com/topics/space-vim has some repositories.

Reply:
Thanks， will add this aliases